server/utils/utils.ts

```diff
@@ -64,8 +64,10 @@ export const pascalCase = (string: string) => camelCase(string).replace(/\w/, s
  */
 export const sentenceCase = (string: string) => Case.sentence(string)
 
-export const lowerCase = (string: string) => Case.lower(string)
-
+export const lowerCase = (string: string) => {
+  const result = Case.lower(string)
+  return result
+}
```

Comment:
Introducing a variable here seems unnecessary -- though this method seems unnecessary in itself... 🤷‍♂️